source/_components/binary_sensor.template.markdown

```diff
@@ -27,7 +28,12 @@ binary_sensor:
         friendly_name: "Sun is up"
         value_template: >-
           {{ state_attr('sun.sun', 'elevation')|float > 0 }}
+        availability_template: >-
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.